src/dissect/dissect.c

```diff
@@ -106,6 +107,7 @@ static int help(void) {
                "     --mkdir              Make mount directory before mounting, if missing\n"
                "     --rmdir              Remove mount directory after unmounting\n"
                "     --discard=MODE       Choose 'discard' mode (disabled, loop, all, crypto)\n"
+               "     --memory             Copy image into memory\n"
```

Comment:
A bit generic, maybe --from-memory or --in-memory? As in "dissect mount in memory"